README.md

```diff
@@ -35,10 +35,13 @@ require('isomorphic-fetch');
 
 fetch('//offline-news-api.herokuapp.com/stories')
 	.then(function(response) {
-		if (response.status >= 400) {
+		if (response.status >= 200 || response.status < 400) {
+			return response.json();
```

Comment:
is this not used?
https://github.com/github/fetch/blob/master/fetch.js#L384
https://github.com/bitinn/node-fetch/blob/master/src/response.js#L41